browser/src/TeamPage/TeamMembers.json

```diff
@@ -0,0 +1,150 @@
+{
+  "gnomadCouncil": [
+    {
+      "name": "Heidi Rehm, Ph.D., FACMG,",
+      "bio": "(MPI) is an Institute Member at the Broad Institute, Chief Genomics Officer at Massachusetts General Hospital (MGH) and Professor of Pathology at Harvard Medical School (HMS). She brings long-standing experience in genomic resource management, clinical sequencing, variant interpretation and gene discovery for rare disease. She co-chairs the gnomAD council and is responsible for jointly overseeing all activities.",
+      "headshotSource": "../assets/images/hrehm_headshot.jpg"
```

Comment:
Here I included the relative local filepath as a property, so that it can be used when mapping